### ISSUE TITLE
Showing psr/log installation error with laravel 5.7

### ISSUE THREAD

**Issue by umairali** [psliwa/image-optimizer#63]
I am trying to install in laravel 5.7 but its showing psr/log error and not supporting latest psr/log package.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for ps/image-optimizer ^1.2 -> satisfiable by ps/imag   e-optimizer[1.2.0].
    - Conclusion: remove psr/log 1.1.0
    - Conclusion: don't install psr/log 1.1.0
    - ps/image-optimizer 1.2.0 requires psr/log 1.0.* -> satisfiable by psr/log[1.0.0, 1.0.1, 1.0.2].
    - Can only install one of: psr/log[1.0.0, 1.1.0].
    - Can only install one of: psr/log[1.0.1, 1.1.0].
    - Can only install one of: psr/log[1.0.2, 1.1.0].
    - Installation request for psr/log (locked at 1.1.0) -> satisfiable by psr/log[1.1.0].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by free6k** [comment]
+1

**Comment by umairali** [comment]
#64 #59

**Comment by sishen007** [comment]
同求

**Comment by psliwa** [comment]
It should be fixed in `1.2.1` version: https://github.com/psliwa/image-optimizer/pull/65. Can you confirm that it doesn't work in `1.2.1` version and give me an error output?
